pages/about.html

```diff
@@ -1,12 +1,62 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en-GB">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <link rel="stylesheet" href="../stylesheets/styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+  </head>
+      <body>
+        <div class="parent">
+            <div class="div1">   
```

Comment:
You shouldn't use class names like `div1`, since they don't communicate what it means at all.  (Also, if you rearrange your `<div>`s or change some of them to `<section>`s or something things would get confusing quickly!)

Reply:
Since it looks like you're making these bands with CSS I'd recommend switching them to a SVG background image (since fundamentally they don't have any real meaning without the styling).